experimental/generation/generator/templates/booleanEntity.en-us.lg.lg

```diff
@@ -5,7 +5,7 @@
 
 # template
 - ```
-# booleanEntity(value)
+${'#'} booleanEntity(value)
```

Comment:
>${'#' [](start = 0, length = 5)

what the wha?

Reply:
Bug in LG that they are working on fixing and this is the work-around until they do.

---
In reply to: [423898362](https://github.com/microsoft/BotBuilder-Samples/pull/2378#discussion_r423898362) [](ancestors = 423898362)